cmd/pebble/main.go

```diff
@@ -62,3 +80,13 @@ func main() {
 		muxHandler)
 	cmd.FailOnError(err, "Calling ListenAndServeTLS()")
 }
+
+func setupCustomDNSResolver(dnsResolverAddress string) {
+	net.DefaultResolver = &net.Resolver{
+		PreferGo: true,
```

Comment:
@ldez Do you know what happens in the case where Go's built-in resolver is not available? It seems like it might quietly fall back to the system resolver and the `net.Dialer` provided below will be ignored. Is that correct? I don't think its a big deal but I'm not sure what systems wouldn't have the Go resolver available.

Reply:
yes if the dialer fails, it quietly fall back to the system resolver.
The Go resolver seems available for unix platform and windows.